src/foundry/client/data/documents/macro.d.mts

```diff
@@ -53,4 +52,9 @@ interface Scope {
    * A Token which is the protagonist of the executed action
    */
   token?: Token;
+
+  /**
+   * @remarks Additional arguments passed as part of the scope
+   */
+  [arg: string]: any;
```

Comment:
Could you make this `unknown` please? It makes it more type safe when accessed even if that's probably pretty uncommon.

I wish I had a simple explanation for when to use `any` versus not but in essence `any` is usually okay when it's part of a more complex bound, like any array or any function. Though even in those cases there's usually workarounds.

So I'm always going to advocate for avoiding it as it's extremely infectious in terms of the soundness hole it creates.